app/data_mapper_setup.rb

```diff
@@ -0,0 +1,7 @@
+require 'data_mapper'
+
+require './app/models/user'
+
+DataMapper::Logger.new($stdout, :debug)
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_#{ENV['RACK_ENV']}")
```

Comment:
Metrics/LineLength: Line is too long. [100/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)